pyproject.toml

```diff
@@ -54,11 +54,10 @@ dependencies = [
     "lxml>=4.5.2",
     "PyRSS2Gen>=1.1",
     "blinker>=1.3",
-    "setuptools>=60.0.5",
     "natsort>=5.1.0",
     "requests>=2.31.0",
     "piexif>=1.0.3",
-    "Babel>=2.12.0",
+    "Babel>=2.17.0",
```

Comment:
Do versions < 2.17 not work? Or why do you bump the requirement?

Reply:
So generally Babel works fine, it's just the unit test that fails? I don't think we want to bump the requirement in that case. This only makes life hard for our users.

Reply:
Well there was some behavioral change and since there is a test for it, it seemed important enough :shrug:

Reply:
The minimum versions in `pyproject.toml` do not guarantee all tests pass. Our tests assume the latest versions of all dependencies. If we changed the minimum version here, then Arch Linux would be unable to package Nikola until they upgraded Babel. But Nikola is working fine with the older version, just might have slightly different output for some date formats/locales. Please undo this change.